andela/wg-baqi
# #166266326 Setup Travis and fix tests

#### What does this PR do?
Sets up Travis and fixing tests
#### Description of Task to be completed?
To have all tests fixed and Travis build passing
#### How should this be manually tested?
Run the tests
```
$python manage.py test
```
Click on the badge and see the build
#### Any background context you want to provide?
Tests were failing due to upgrading to django 2
#### What are the relevant pivotal tracker stories?
#166266326
#### Screenshots (if appropriate)
<img width="1171" alt="Screenshot 2019-05-29 at 23 39 49" src="https://user-images.githubusercontent.com/31995387/58611979-a593d380-82b9-11e9-8707-baa1e5e1ee2c.png">
